src/Task/Git/CommitMessage.php

```diff
@@ -135,6 +138,9 @@ public function run(ContextInterface $context): TaskResultInterface
             );
         }
 
+        if ($this->isMergeCommit($commitMessage)) {
```

Comment:
Would it make sense to add a configuration flag like `skip_on_merge_commit` (default `true`) and do the validation at the top of the task - before any other task runs?

That way people could opt-in on checking commit messages on merges (which might be a bit odd, but someone might use it.) Setting this by default to true is a bc break somehow, but I think it makes total sense to do so. People can easily re-activate it by config.

WDYT?

Reply:
Sounds like a sensible idea, although wouldn't a default of `false` be better, as then no behaviour will change unless you set it. Although as you say, using it on merges is a bit odd and probably the exception.